qstash/workflow/quickstarts/vercel-nextjs.mdx

```diff
@@ -60,6 +62,23 @@ export const POST = serve(
   }
 )
 ```
+</Tab>
+
+<Tab title="With request object">
+```typescript app/api/workflow/route.ts
+
+import { serve } from "@upstash/qstash/nextjs"
+
+export const POST = (req: Request) => {
+  // do something with native request object
+
+  return serve(async (context) => { 
+    // Your workflow steps
+   })
```

Comment:
this won't work as the serve method is not passed the req at all.

should have been like this:
```javascript
export const POST = (req: Request) => {
  // do something with native request object

  const handler = serve(async (context) => { 
    // Your workflow steps
   })
  return await handler(req)
}

```

Reply:
resolving this thread as it looks like this code snippet is removed